modules/ROOT/pages/import-api-specification-maven.adoc

```diff
@@ -0,0 +1,63 @@
+= Importing an API Specification from Maven
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification in a Maven repository into a new Mule Project, or an existing one.
+
+This feature is possible through API Sync. If you are using an older version of Mule Runtime (4.1.3 and below), or if you don't want to use this new design paradigm and would still prefer to import an API specification that's not in Exchange, follow the xref:studio::import-api-specification-design-center.adoc[Importing an API Specification from a Local File] steps.
+
+== Importing an API From Maven Into a New Project
+
+. In the top task bar, select *File*, then *New*, and then *Mule Project*.
+. Type a name for your project.
+. Select a Mule runtime engine version 4.1.4 or later.
+. In the *API Specification* section, select the *From Exchange or Maven* tab, click the plus icon, and select *From Maven*. +
```

Comment:
the plus icon > the Add icon. If it is not clear enough, you can insert the icon inline